v1/image.go

```diff
@@ -15,57 +15,44 @@
 package v1
 
 import (
-	"io"
-
 	"github.com/google/go-containerregistry/v1/types"
 )
 
 // Image defines the interface for interacting with an OCI v1 image.
 type Image interface {
-	// FSLayers returns the ordered collection of filesystem layers that comprise this image.
-	// The order of the list is most-recent first, and oldest base layer last.
-	FSLayers() ([]Hash, error)
-
-	// DiffIDs returns the ordered list of uncompressed layer hashes (matches FSLayers).
+	// Layers returns the ordered collection of filesystem layers that comprise this image.
 	// The order of the list is most-recent first, and oldest base layer last.
-	DiffIDs() ([]Hash, error)
-
-	// ConfigName returns the hash of the image's config file.
-	ConfigName() (Hash, error)
+	Layers() ([]Layer, error)
 
 	// BlobSet returns an unordered collection of all the blobs in the image.
 	BlobSet() (map[Hash]struct{}, error)
 
-	// Digest returns the sha256 of this image's manifest.
-	Digest() (Hash, error)
-
 	// MediaType of this image's manifest.
 	MediaType() (types.MediaType, error)
 
-	// Manifest returns this image's Manifest object.
-	Manifest() (*Manifest, error)
-
-	// RawManifest returns the serialized bytes of Manifest()
-	RawManifest() ([]byte, error)
+	// ConfigName returns the hash of the image's config file.
+	ConfigName() (Hash, error)
```

Comment:
I'd expect this to be "ConfigDigest", I think.

Reply:
#63 made me realize this was already called `ConfigName`, and I just misparsed the diff. Feel free to ignore this, though we now have some empirical evidence supporting `ConfigDigest` instead :)